exec.in

```diff
@@ -1,10 +1,13 @@
-#!/bin/sh
+#!/bin/bash
+export O_PATH="$PATH"
 export PATH="@CMAKE_INSTALL_PREFIX@/bin:@RUSTUP_LOCATION@/.cargo/bin:$PATH"
 export PKG_CONFIG="pkgconf --static"
 export PKG_CONFIG_LIBDIR="@MINGW_INSTALL_PREFIX@/lib/pkgconfig"
 export RUSTUP_HOME="@RUSTUP_LOCATION@/.rustup"
 export CARGO_HOME="@RUSTUP_LOCATION@/.cargo"
 export LD_PRELOAD="@MALLOC@"
 export CCACHE_DIR="@MINGW_INSTALL_PREFIX@/ccache"
+export LLVM_PROFILE_FILE="/dev/null"
```

Comment:
Why this is necessary? `LLVM_PROFILE_FILE` var only exported when  in `GEN` or `CSGEN`. When `OFF` or `USE` it will never export